pkg/resources/management.cattle.io/v3/globalrole/validator_test.go

```diff
@@ -545,6 +545,146 @@ func TestAdmit(t *testing.T) {
 			},
 			allowed: false,
 		},
+		{
```

Comment:
Nit:
If you can add two tests with invalid rules (rules without verbs should do it), one for namespacedRules and one for Rules, I would appreciate it.